server/app/assets/stylesheets/uswds/_uswds-theme-custom-styles.scss

```diff
@@ -24,3 +24,8 @@ See the radio-option below as an example.
 .cf-radio-option:focus-within {
   @include u-border('blue-40v');
 }
+
+/* Make the file input element for program image upload fill the area it's given. */
+.usa-file-input {
```

Comment:
Good question. We can't set it directly in `FileUploadViewStrategy` because of how the USWDS component works. In `FileUploadViewStrategy`, the only divs we create are one classed as `usa-form-group` and another classed as `usa-file-input`. But, USWDS actually creates a lot more items than just those two (see below). I tried setting an ID on the `<input>` element created in `FileUploadViewStrategy`, but that ID turns out to apply to the element with the `usa-file-input__input` class, but we actually need the `max-width` override on the element with the `usa-file-input` class.

I updated the style to be `#program-image-upload-file-input .usa-file-input` so that it only modifies file inputs within the program image upload div and it looks like that works! And shouldn't affect other file uploads.

```
<div class="usa-file-input usa-file-input--disabled">
  <div class="usa-file-input__target">
    <div class="usa-file-input__box"></div>
    <div class="usa-file-input__instructions" aria-hidden="true">
      <span class="usa-file-input__drag-text">Drag file here or</span> 
      <span class="usa-file-input__choose">choose from folder</span>
    </div>
    <input id="program-image-upload-file-input" type="file" name="file" class="usa-file-input__input" aria-describedby="file-input-hint-0 file-input-hint-1 " accept="image/jpeg,image/png" disabled="" aria-label="Drag file here or choose from folder">
  </div>
</div>
```